modules/ossm-using-discoveryselectors-scope-service-mesh.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-discoveryselectors-scope-service-mesh_{context}"]
+= Using DiscoverySelectors to scope a Service Mesh
+:context: ossm-discoveryselectors-scope-service-mesh
+
+If you know which namespaces to include in the {SMProductShortName}, configure `discoverySelectors` during or after installation by adding the required selectors to the MeshConfig resource of {istio}. For example, configure {istio} to discover only namespaces labeled `istio-discovery=enabled`.
```

Comment:
```suggestion
If you know which namespaces to include in the {SMProductShortName}, configure `discoverySelectors` during or after installation by adding the required selectors to the `meshConfig.discoverySelectors` section of the `{istio}` resource. For example, configure {istio} to discover only namespaces labeled `istio-discovery=enabled`.
```
Apologies for only seeing this now - the reference ot "MeshConfig resource..." looks like an upstream reference, so it would be more accurate to describe the configuration value changed in the `Istio` resources which is part of the product (as shown in step 2).

Reply:
Made this change. Thanks!